Desktop/data/importers/excel/__freexl.h

```diff
@@ -0,0 +1,736 @@
+// fetch from: https://www.gaia-gis.it/fossil/freexl/dir?ci=tip
```

Comment:
My commit to use conan was not complete: I just renamed freexl.h to __freexk.h, to be sure that this file won't be used. I did not remove it, because I wanted Shun to test first whether the conan version compiles and works also on Windows. The CMakefile.txt, Install.cmake and Libraries.cmake must be changed also.

Reply:
But I also dont see why this file would be needed without conan?

I know I did something like this with readstat but that was to make it play nice with c++ I think

Reply:
Because when using cmake to reference `freexl.h` in the msys2(Rtools) include path, it will automatically link some other standard libraries, which will conflict with the msvc compiler and linker during compilation. However, once we use conan, we do not need to introduce this header.